rfcs/061-content-api-next-steps/README.md

```diff
@@ -0,0 +1,414 @@
+
+The active endpoints are:
+- https://api.wellcomecollection.org/content/v0/events
+- https://api.wellcomecollection.org/content/v0/events/[eventId]
+
+Other references:  
+[Original suggestions for future versions of Content API](https://github.com/wellcomecollection/docs/blob/main/rfcs/055-prismic-api/future-versions.md) (includes suggestions of filters and such for events)
+
+Notes:  
+- This document will be using the term “Scheduled” to talk about “children” events.
+- This API only serves Prismic `Events` content types. Please note that it doesn't fetch anything from the `Events Series` content type. The name of a series does get considered at the display level as it gets displayed on the result card along with a “Part of” prefix, but gets pulled from the `Event` data.
+- Some of these challenges also affect the `Articles` endpoint. If that is the case, it will be indicated.
+
+
+## Challenges encountered and decisions made  
+
+### Scheduled events
+This [Github PR](https://github.com/wellcomecollection/content-api/issues/93#issue-2088674788) described this and applied a solution that is subject to change.
+
+#### Challenge
+There is no way to create an event hierarchy at the moment. In reality, some events come as a set (they become “Scheduled” events) and could use such a thing. The solution adopted by the team is to create a “Parent” event first, which contains the general information including promotional images. Then, “Scheduled” events are added to its “Scheduled” tab:
+
+![](./schedule-on-event.png)
+
+These Scheduled events are, in reality, sibling events, as, as we said, there is no hierarchy within the Event content type. They are created only to be listed in the Parent event:
+
+As they contain minimal other information (lacking images, description...), the Scheduled events are not to be seen individually, nor displayed in listing pages. This is done by tagging the event with delist.
+
+The problem with search is that the Scheduled events contain important information such as date/time and access needs (interpretations). A Parent event would have no interpretation, but it might have a Scheduled event that is a BSL tour, and another one that is an Audio-described one.
+
+The parent points to its children by ID, the child has no knowledge of what its parent is
+#### Decision
+
+- Scheduled events must be findable/filterable based on their interpretation, audience, etc... which are not set on the parent
+- The API returns the Parent event, not the Scheduled events to avoid duplication and/or evenst being listed without image or description
+- Basically we want to be able to match on a Scheduled event but return its parent in the API response 
+
+Therefore, we need the Parent event document to contain all the filterable values for its Scheduled events.
```

Comment:
The `display` object should also have the Scheduled events' data (eg. audience, format), so as to display it on the search result card